packages/lodestar-beacon-state-transition/src/lightclient/sync/update.ts

```diff
@@ -0,0 +1,71 @@
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
```

Comment:
We should maybe make a new package for this.
Isn't really part of the "beacon-state-transition"

Reply:
Yeah, totally makes sense.